roda-core/roda-core/src/main/resources/config/roda-core.properties

```
@@ -621,4 +621,37 @@ core.user_registration.disabled = false
 #
 # Status: experimental
 ##########################################################################
-core.storage.legacy.implementation.enabled = false
\ No newline at end of file
+core.storage.legacy.implementation.enabled = false
+
+##########################################################################
+# External authentication settings
+#
+# core.authorization.external.group_mapping determines whether to use the
+# CAS group mapping feature or not.
+#
+# Use core.authorization.external.attribute to define which CAS attribute
+# will be checked  to find the groups that a CAS user is part of. If left
+# undefined, will default to 'memberOf'
+#
+# To map CAS groups to a RODA groups, first define a new mapping with the
+# following property:
+# core.authorization.external.mappings[] = sampleCasMapping1
+# Then, define the regex for the external groups that will be mapped to
+# RODA groups:
+# core.authorization.external.mapping.sampleCasMapping1.external.group = sampleCasGroup\\d+
+# Finally, define the RODA groups that the CAS users in CAS groups that match
+# that regex will be put in:
+# core.authorization.external.mapping.sampleCasMapping1.internal.groups[] = sampleRodaGroup1
+# core.authorization.external.mapping.sampleCasMapping1.internal.groups[] = users
+##########################################################################
+# CAS group mapping toggle
+core.authorization.external.group_mapping = true
+
+# CAS user groups attribute name
+core.authorization.external.attribute = memberOf
```

Leave it commented to be activated upon request

```suggestion
#core.authorization.external.attribute = memberOf
```